Makefile

```diff
@@ -23,6 +23,19 @@ in: inplace # just a shortcut
 inplace:
 	$(PYTHON) setup.py build_ext -i
 
+dev-meson:
```

Comment:
```suggestion
dev-meson:
    # Temporary hack to try the experimental meson build. Once meson is accepted
    # as the default build tool, this will go away.
```